### PR TITLE
Add animation to carrousel

### PR DESCRIPTION
**Pull request by antsgar**
**Description**

Added animation to cards carrousel when you swap cards.

Jira link:

**Type of change**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Passes test case
- [x] UI components visually reviewed on desktop view
- [x] UI components visually reviewed on mobile view
- [ ] Other (please explain)

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [ ] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

### CODE REVIEW

**Review by SayakaOno — APPROVED** [review]
The animation looks nice :)

File: packages/webapp/src/components/CardsCarrousel/styles.module.scss [hunk 1]
```diff
@@ -39,22 +39,49 @@
   }
 }
 
+@keyframes card-to-front {
```
Comment (kathyavini) [inline packages/webapp/src/components/CardsCarrousel/styles.module.scss:42]: This animation is defined but not used.

File: packages/webapp/src/components/CardsCarrousel/styles.module.scss [hunk 2]
```diff
@@ -39,22 +39,49 @@
...
   @media only screen and (min-width: 1024px) {
     padding-right: 20px;
     padding-left: 24px;
   }
 }
 
+.inactiveCardButton {
+  height: 100%;
+  width: calc(100% - 2px);
+  overflow: hidden;
```
Comment (kathyavini) [inline packages/webapp/src/components/CardsCarrousel/styles.module.scss:81]: For me this cuts off the icons on the left. Is that the intended behaviour?

Screenshot 2023-11-06 at 11 17 11 AM

**Review by kathyavini — COMMENTED** [review]
It's subtle, but really nice!!

I just had two small questions which I left in-line.

Also (and it's not from this PR so please feel free to direct elsewhere) -- in looking really closely at the icons, I noticed the profit-loss icon is quite pixelated when it's on the active card. It looks like the SVG set might need replacing with something a little cleaner, is that already on your radar?

Screenshot 2023-11-06 at 11 23 06 AM

File: packages/webapp/src/components/CardsCarrousel/styles.module.scss (hunk 2 above)
Comment (antsgar) [reply packages/webapp/src/components/CardsCarrousel/styles.module.scss:81]: It was intended to cut them off in mobile when the cards are more overlapped, but they shouldn't look like this in desktop! Will take a look

**Comment by antsgar** [comment]
@kathyavini I addressed your comments! I do see the profit loss icon pixelated, but I think I might have to swap SVGs with PNGs or maybe export only the icon as SVG to fix it. I'll create a ticket and take a look later

**Review by kathyavini — APPROVED** [review]

**Review by SayakaOno — APPROVED** [review]

**Pull request merged**